test/plugins/scenarios/goframe/plugin.yml

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+entry-service: http://${HTTP_HOST}:${HTTP_PORT}/provider
+health-checker: http://${HTTP_HOST}:${HTTP_PORT}/health
+start-script: ./bin/startup.sh
+framework: github.com/gogf/gf
+export-port: 8080
+support-version:
+  - go: 1.18
```

Comment:
the goframe have multiple versions, please update it to `https://github.com/gogf/gf/releases`. 
In the test case, you still need to update these two files:
1. [plugins.md](https://github.com/apache/skywalking-go/blob/main/docs/en/agent/support-plugins.md): please update the support plugins documentation.
2. [plugin test](https://github.com/apache/skywalking-go/blob/main/.github/workflows/plugin-tests.yaml#L77-L104): please add this plugin test, otherwise the plugin test would not be triggered.